projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1237,6 +1441,58 @@
 
   "biome.quark.glimmering_weald": "微光荒野",
 
+  "tag.item.c.storage_blocks.apple": "苹果存储方块",
+  "tag.item.c.storage_blocks.bamboo": "竹存储方块",
+  "tag.item.c.storage_blocks.beetroot": "甜菜根存储方块",
+  "tag.item.c.storage_blocks.blaze_rod": "烈焰棒存储方块",
+  "tag.item.c.storage_blocks.cactus": "仙人掌存储方块",
+  "tag.item.c.storage_blocks.carrot": "胡萝卜存储方块",
+  "tag.item.c.storage_blocks.charcoal": "木炭存储方块",
+  "tag.item.c.storage_blocks.chorus_fruit": "紫颂果存储方块",
+  "tag.item.c.storage_blocks.cocoa_beans": "可可豆存储方块",
+  "tag.item.c.storage_blocks.glow_berries": "发光浆果存储方块",
+  "tag.item.c.storage_blocks.golden_apple": "金苹果存储方块",
+  "tag.item.c.storage_blocks.golden_carrot": "金胡萝卜存储方块",
+  "tag.item.c.storage_blocks.gunpowder": "火药存储方块",
+  "tag.item.c.storage_blocks.leather": "皮革存储方块",
+  "tag.item.c.storage_blocks.nether_wart": "下界疣存储方块",
+  "tag.item.c.storage_blocks.potato": "马铃薯存储方块",
+  "tag.item.c.storage_blocks.rabbit_hide": "兔子皮存储方块",
+  "tag.item.c.storage_blocks.stick": "木棍存储方块",
+  "tag.item.c.storage_blocks.sugar_cane": "甘蔗存储方块",
+  "tag.item.c.storage_blocks.sweet_berries": "甜浆果存储方块",
+
+  "tag.item.quark.ancient_logs": "远古木",
+  "tag.item.quark.azalea_logs": "杜鹃木",
+  "tag.item.quark.bamboo_logs": "竹",
+  "tag.item.quark.big_harvesting_hoes": "可以范围收获的锄",
+  "tag.item.quark.blossom_logs": "花木",
+  "tag.item.quark.corundum": "刚玉",
+  "tag.item.quark.crab_tempt_items": "螃蟹的食物",
+  "tag.item.quark.crystal_lamp": "水晶灯",
+  "tag.item.quark.cosmetic_anvil_items": "铁砧型装饰性升级件",
+  "tag.item.quark.framed_glass_panes": "镶框玻璃板",
+  "tag.item.quark.framed_glasses": "镶框玻璃",
+  "tag.item.quark.glow_shroom_feedables": "发光菇 - 石精灵的食物",
+  "tag.item.quark.hedges": "树叶篱",
+  "tag.item.quark.hollow_logs": "空心的原木",
+  "tag.item.quark.ladders": "梯子",
+  "tag.item.quark.parrot_feed": "鹦鹉的食物",
+  "tag.item.quark.pipes": "管道",
+  "tag.item.quark.posts": "木杆",
+  "tag.item.quark.revertable_chests": "可以转换成原版箱子的箱子",
+  "tag.item.quark.revertable_trapped_chests": "可以转换成原版陷阱箱的陷阱箱",
+  "tag.item.quark.runes": "符文",
+  "tag.item.quark.runes_lootable": "可以作为战利品获得的符文",
+  "tag.item.quark.seed_pouch_holdable": "种子袋可以容纳的物品",
+  "tag.item.quark.shards": "玻璃碎片",
+  "tag.item.quark.stained_framed_glass_panes": "染色镶框玻璃板",
+  "tag.item.quark.stained_framed_glasses": "染色镶框玻璃",
+  "tag.item.quark.stone_tool_materials": "石质工具材料",
+  "tag.item.quark.stools": "凳子",
+  "tag.item.quark.vertical_slabs": "竖直台阶",
+  "tag.item.quark.wooden_vertical_slabs": "木质竖直台阶",
+
   "advancements.quark.influence.title": "影响者",
   "advancements.quark.influence.description": "使用蜡烛来影响矩阵附魔台的附魔效果",
```

```suggestion
  "advancements.quark.influence.description": "使用蜡烛来影响矩阵附魔台对特定种类附魔的'看法'",
```
这个建议可用可不用，主要是与上文“意见领袖”配合